### PR TITLE
Add team Readme documents to Sprint Teams page

### PR DESCRIPTION
**Pull request by aduth**
**Why**: So that a reader can learn more about the team in the context of reading about each of our teams, and so that someone seeking a Readme has a centralized listing of them.

**Screenshot:**

image

### CODE REVIEW

File: _data/sprint_teams.yml [hunk 1]
```diff
@@ -4,19 +4,22 @@
...
   slack_channel_url: https://gsa-tts.slack.com/archives/C03N6KQBAKS
+  readme: https://docs.google.com/document/d/1NQluZ_oo58ER4T49XzCvowdmXEYbOpZf-xS6MoT2Yng/edit
 
 - name: Grace
   namesake_markdown: "[Admiral Grace Hopper](https://en.wikipedia.org/wiki/Grace_Hopper), a computer scientist and navy officer"
   focus_area: KYC/Non-biometric verification
   slack_channel_name: login-team-grace
   slack_channel_url: https://gsa-tts.slack.com/archives/C0184P9SCJ0
   slack_group_name: login-grace
+  readme: https://docs.google.com/document/d/1x_IPoYc_kavE6Rw58leIckp85Js7fKPy_NebpqC3iN0/edit
```
Comment (zachmargolis) [inline _data/sprint_teams.yml:22]: thanks for adding all the readmes!

**Review by zachmargolis — APPROVED** [review]
LGTM

**Comment by zachmargolis** [comment]
also! looks like the PrivateEye link JS isn't running in your preview screenshot, but it works as expected in my browser on the [preview branch](https://federalist-7d0b2b76-42fc-4df1-9825-8c3cd77e0a15.app.cloud.gov/preview/18f/identity-handbook/aduth-team-readme/articles/sprint-teams.html)

Screen Shot 2022-09-12 at 7 42 18 AM

**Comment by aduth** [comment]
> also! looks like the PrivateEye link JS isn't running in your preview screenshot, but it works as expected in my browser on the [preview branch](https://federalist-7d0b2b76-42fc-4df1-9825-8c3cd77e0a15.app.cloud.gov/preview/18f/identity-handbook/aduth-team-readme/articles/sprint-teams.html)

Hm, strange. It looks like there might be some race condition between the Jekyll and JavaScript compilation, because it hadn't copied over the transpiled `setup.js` when run locally. Saving the JS file again triggered Jekyll to regenerate and then the lock shows up correctly.

As you pointed out, I don't think it'll be an issue for production site builds, but it's probably worth digging into for local development.

**Pull request merged**